src/audio/buffer.c

```diff
@@ -31,6 +31,7 @@ struct comp_buffer *buffer_alloc(uint32_t size, uint32_t caps, uint32_t flags, u
 {
 	struct comp_buffer *buffer;
 	struct comp_buffer __sparse_cache *buffer_c;
+	void *stream_addr;
```

Comment:
commit message: `buffer_alloc()` not `buffer_init()`

Reply:
fixed in V2